book/website/_config.yml

```diff
@@ -58,6 +58,7 @@ sphinx:
   config                    :  # key-value pairs to directly over-ride the Sphinx configuration
     language                : en
     html_show_copyright     : false
+    html_extra_path         : ["netlify"]
```

Comment:
We already have redirects declared in `netlify.toml`. Can these also go there?

Reply:
Oh, right, I missed those! Yes, these rules can go into `netlify.toml` or the existing ones can move into `_redirects`. Maybe better to keep them all in `netlify.toml`: it's more verbose but that makes it a little more self-explanatory. What do you think @JimMadge?

Reply:
Yeah I think the `toml` file makes most sense for now :+1:.